app/util/appUtil.js

```diff
@@ -0,0 +1,153 @@
+const { ApiPromise, WsProvider } = require('@polkadot/api')
```

Comment:
This is a really random app util with multiple concerns. Consider breaking into a couple of different files related to the `api` and the `auth`